### DISCUSSION TITLE
Can't install @walletconnect/web3-provider

### DISCUSSION THREAD

**Discussion by markibanez** [WalletConnect/walletconnect-monorepo#742] [Developer Support] [answered]
Attempting to install via command `npm i @walletconnect/web3-provider` on Windows 10 with Node 16

I'm getting the following error:

```
PS D:\Github\goingup-xyz\web-app> npm i @walletconnect/web3-provider
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
npm ERR! errno -4058
npm ERR! enoent An unknown git error occurred
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Mark\AppData\Local\npm-cache\_logs\2022-01-25T10_19_00_644Z-debug-0.log
```

Thanks in advance

**Comment by markibanez** [comment, marked as answer]
Seems the issue has resolved itself

**Reply by Khan3930** [reply]
hi i also face this problem how can i solve

**Reply by Mehrtens** [reply]
I have same issue, other packages install fine, just not this one. Has anybody figured this out? It seems I am missing a dependency or something.

**Reply by paulflo150** [reply]
has anyone figured this out?

**Comment by Birdland998** [comment]
Try this https://stackoverflow.com/questions/52522565/git-is-not-working-after-macos-update-xcrun-error-invalid-active-developer-pa
